tests/__main__.py

```diff
@@ -39,23 +52,27 @@ def convert_argv(argv):
             help = True
         args.append(arg)
 
-    # We make the "executable" a single arg because unittest.main()
-    # doesn't work if we split it into 3 parts.
-    cmd = [sys.executable + ' -m unittest']
-    if not modules and not help:
-        # Do discovery.
-        if quick:
-            start = os.path.join(TEST_ROOT, 'ptvsd')
-        elif sys.version_info[0] != 3:
-            start = os.path.join(TEST_ROOT, 'ptvsd')
-        else:
-            start = PROJECT_ROOT
-        cmd += [
-            'discover',
-            '--top-level-directory', PROJECT_ROOT,
-            '--start-directory', start,
-        ]
-    return cmd + args
+    if runtests:
+        # We make the "executable" a single arg because unittest.main()
+        # doesn't work if we split it into 3 parts.
+        cmd = [sys.executable + ' -m unittest']
+        if not modules and not help:
+            # Do discovery.
+            if quick:
+                start = os.path.join(TEST_ROOT, 'ptvsd')
+            elif sys.version_info[0] != 3:
+                start = os.path.join(TEST_ROOT, 'ptvsd')
```

Comment:
This is identical to the branch above - should it be a single branch with an `or`?

Reply:
It doesn't really matter.  Personally I prefer to keep the distinct cases separate, even if they do the same thing currently.